docs/developer/building/requirements.rst

```diff
@@ -0,0 +1,32 @@
+Requirements
+============
+
+In order to build MetalK8s we rely and third-party tools, some of them are
+mandatory, others are optional.
+
+Mandatory
+---------
+
+- `Python <https://www.python.org/>`_ 3.6 or higher: our buildchain is
```

Comment:
buildchain *and* BDD tests

Reply:
This requirements is only about the build chain.